R/aes-colour-fill-alpha.r

```diff
@@ -1,7 +1,48 @@
 #' Colour related aesthetics: colour, fill and alpha
 #'
-#' This page demonstrates the usage of a sub-group
-#' of aesthetics: colour, fill and alpha.
+#' These aesthetics parameters change the colour (`colour` and `fill`) and the
+#' opacity (`alpha`) of geom elements on a plot. Almost every geom has either
+#' colour or fill (or both), as well as can have their alpha modified.
+#' Modifying colour on a plot is useful way to enhance the presentation of data,
+#' often especially when a plot graphs more than two variables.
+#'
+#' @section Colour and fill:
+#'
+#' Colours and fills can be specified in the following ways:
+#' * A name, e.g., "red". R has 657 built-in named colours, which can be
+#' listed with colours().
```

Comment:
Put `colours()` in back quotes, and maybe also in square brackets to create a link. Please try if this works before committing this change.